app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventInitial/EventInitialActivity.java

```diff
@@ -945,4 +949,126 @@ public void showEventWasDeleted() {
         showToast(getString(R.string.event_was_deleted));
         finish();
     }
+
+    @Override
+    public void setGeometryModel(CoordinateViewModel geometryModel) {
+        setGeometryCallback(geometryModel);
+        binding.geometry.setItem(geometryModel);
+    }
+
+    private void setGeometryCallback(CoordinateViewModel geometryModel) {
+        currentGeometryModel = geometryModel;
+        geometryModel.setCallback(new FieldUiModel.Callback() {
+            @Override
+            public void onNext() {
+                //Not used
+            }
+
+            @Override
+            public void showDialog(@NotNull String title, @org.jetbrains.annotations.Nullable String message) {
+                //Not used
+            }
+
```

Comment:
should we use a specific interface? empty methods like these break interface segregation  @andresmr???

Reply:
Note that this is not in the form. This an individual usage of the coordinate layout in the event initial screen. I would not change more of this activity until we move all of it to this "first section" of the event form. We can create a ticket for this as soon as possible.